src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Utilities/FilePathUtilties.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Utilities
+{
+    internal static class FilePathUtilities
```

Comment:
I used this to get relative path in TargetFrameworkValueProvider, I wasn't aware of the existing utility method that you pointed out in another comment. I'll remove these files.